csrankings.csv

```diff
@@ -1352,6 +1352,7 @@ Anisoara Calinescu,University of Oxford,http://www.cs.ox.ac.uk/people/ani.caline
 Anita Sarma,Oregon State University,http://eecs.oregonstate.edu/people/sarma-anita,O4TGxGMAAAAJ
 Anita Wasilewska,Stony Brook University,http://www3.cs.stonybrook.edu/~anita,NOSCHOLARPAGE
 Anja Feldmann [INF],Max Planck Society,https://www.mpi-inf.mpg.de/inet/people/anja-feldmann,mSK3340AAAAJ
+Anja Lehmann,Plattner Institute,https://hpi.de/das-hpi/personen/professoren/prof-dr-anja-lehmann.html,zH3fwYoAAAAJ

```

Comment:
Typo in affiliation.

Reply:
I already resolved this issue in this commit:
https://github.com/emeryberger/CSrankings/pull/2691/commits/7359505a08a4802b80b8e5f1ba440b8d22b449dd